mindmeld-ui/README.md

```diff
@@ -0,0 +1,31 @@
+# Mindmeld UI
+
+## Quick Start
+
+This is a React application, you can run it by following the next steps.
+
+### 1. Install the dependencies
+
+In order to run the app, you need to install it's dependencies. You can simply run:
+
+```
+npm install
+```
+
+### 2. Run the app
+
+To run the app simply do:
+
+```
+npm start
+```
+
+### 3. Setup configurations (Optional)
+
+At the root directory, you can add a `.env` file that includes your configurations. This file should look like this:
+
+```
+REACT_APP_GOOGLE_KEY = <YOUR_GOOGLE_KEY> # Set this if you want to use Google Cloud's Speech API (as opposed to Chromes native ASR).
+REACT_APP_MAX_ALTERNATIVES = <MAX_ALYTERNATIVES> # Set this if you want to control how many responses you get from the ASR.
+REACT_APP_MINDMELD_URL = <MINDMELD_URL> # Set this if you need to point the UI to point the UI to a URL different than the default 'http://127.0.0.1:7150/parse'
```

Comment:
I suggest that we use a better namespace than `REACT_APP` for these env vars. Maybe `MINDMELD_UI` and `MINDMELD_UI_NLP_URL` for REACT_APP_MINDMELD_URL`

Reply:
React requires to prefix env variables with `REACT_APP_` [here](https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env). We can change that, but we would need to `eject`, and that's something I'm trying to avoid.